packages/canary-integration/src/__tests__/capability.test.ts

```diff
@@ -315,4 +318,56 @@ describe('CACAO Integration test', () => {
       expect(doc.metadata.controllers).toEqual([`did:pkh:eip155:1:${wallet.address}`])
     }, 30000)
   })
+
+  describe('Ceramic dids instance with capability/parent', () => {
+    test('can update with streamId in capability', async () => {
+      ceramic.did = didKeyWithParent
+      // Create a determinstic tiledocument owned by the user
+      const deterministicDocument = await TileDocument.deterministic(ceramic, {
+        deterministic: true,
+        family: 'testCapabilities1',
+      })
+      const streamId = deterministicDocument.id
+      const didKeyWithCapability = await addCapToDid(wallet, didKey, `ceramic://${streamId.toString()}`)
+      ceramic.did = didKeyWithCapability
+
+      await deterministicDocument.update({ foo: 'bar' }, null, {
+        asDID: didKeyWithCapability,
```

Comment:
@zachferland Do I miss something or `asDID` is not necessary here?

Reply:
yup! removed